R/driver.R

```diff
@@ -342,11 +367,13 @@ warn_data_injects <- function() {
   # Are any chunks are using temp-data-inject data that are also available to them through the data system?
   ci <- chunk_inputs(find_chunks(include_disabled = FALSE)$name)
   chunk_outputs(find_chunks(include_disabled = FALSE)$name) %>%
-    rename(upstream_chunk = name) ->
+    rename(upstream_chunk = name) %>%
+    mutate(output = output) ->
```

Comment:
?

Reply:
Good question.. it has been too long for me to remember exactly but I was getting some odd error message something along the lines of join looses tibble meta data or something.

I think I tried this change thinking it could be something like [Common Fails - mutate forces copy](https://github.com/JGCRI/gcamdata/wiki/Common-Fails-and-How-to-Fix-Them#error-in-add_title---not-allowed-to-overwrite-current-title-).

But I think ultimately the fix was the change in the line below.  I removed this change and verified everything still works for me with no odd error messages so I will push that up.